plotter/contour.go

```diff
@@ -122,6 +124,10 @@ const naive = false
 
 // Plot implements the Plot method of the plot.Plotter interface.
 func (h *Contour) Plot(c draw.Canvas, plt *plot.Plot) {
+	if h.Min > h.Max {
+		panic("contour: negative Z range")
```

Comment:
instead of `negative Z range` we could perhaps say `invalid Z range` (I could certainly imagine a contour which represents, say, a negative altitude for a seafloor map) ?

Reply:
The string reflects the code used in `HeatMap`. We can change it, but I think in another PR.